packages/mdc-tabs/tab-bar-scroller/mdc-tab-bar-scroller.scss

```diff
@@ -52,16 +52,8 @@ $mdc-tab-bar-scroller-indicator-width: 48px;
 
     @include mdc-theme-prop(color, text-secondary-on-light);
```

Comment:
Can you replace all occurrences of `-on-light` with `-on-background` in this file?